starlingcaptureapi/file_util.py

```diff
@@ -56,3 +58,48 @@ def digest_sha256(self, file_path):
                 hasher.update(byte_block)
             return hasher.hexdigest()
         # TODO: handle error (image not found, etc.)
+
+    def cidv1(self, file_path):
+        """Generates the CIDv1 of a file, as determined by ipfs add.
+
+        Args:
+            file_path: the local path to a file
+
+        Returns:
+            CIDv1 in the canonical string format
+        """
```

Comment:
So it's clear caller needs to catch:
```suggestion

        Raises:
            Exception if errors are encountered during processing
        """
```